agent/fw_wordpress.c

```diff
@@ -569,16 +615,71 @@ NR_PHP_WRAPPER(nr_wordpress_apply_filters) {
       NR_PHP_WRAPPER_CALL;
     }
 
+    zval** retval_ptr = NR_GET_RETURN_VALUE_PTR;
     nr_wordpress_name_the_wt(tag, retval_ptr TSRMLS_CC);
   } else {
     NR_PHP_WRAPPER_CALL;
   }
+#endif //OAPI
 
   nr_php_arg_release(&tag);
 }
 NR_PHP_WRAPPER_END
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+     && !defined OVERWRITE_ZEND_EXECUTE_DATA
+NR_PHP_WRAPPER(nr_wordpress_handle_tag_stack_after) {
+  (void)wraprec;
+  /* using nr_php_get_user_func_arg() so that we don't perform another copy
+   * when all we want to do is check the string length */
+  zval* tag = nr_php_get_user_func_arg(1, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
+  if (1 == nr_php_is_zval_non_empty_string(tag)
+      && (0 != NRINI(wordpress_hooks))) {
+      char* cleaned_tag = nr_stack_pop(&NRPRG(wordpress_tags));
+      nr_free(cleaned_tag);
+  }
+}
+NR_PHP_WRAPPER_END
+
+NR_PHP_WRAPPER(nr_wordpress_apply_filters_after) {
+  /* using nr_php_get_user_func_arg() so that we don't perform another copy
+   * when all we want to do is check the string length */
+  zval* tag = nr_php_get_user_func_arg(1, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
+  if (1 == nr_php_is_zval_non_empty_string(tag)) {
+    zval** retval_ptr = NR_GET_RETURN_VALUE_PTR;
+    nr_wordpress_name_the_wt(tag, retval_ptr TSRMLS_CC);
+  }
+
+  nr_wordpress_handle_tag_stack_after(NR_SPECIALFNPTR_ORIG_ARGS);
```

Comment:
NR_SPECIALFNPTR_ORIG_ARGS will resolve to:
```
#define NR_SPECIALFNPTR_ORIG_ARGS \
  wraprec, auto_segment, execute_data, func_return_value
```
execute_data and func_return_value will be NULL.  Make sure `nr_wordpress_handle_tag_stack_after` is okay with that.

Reply:
fixed in [79c7ee8](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/79c7ee84f4d33549eb345c94afaea0a58874d33d)